src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -34,23 +34,40 @@ export class RepresentationMetadata {
    *
    * `@ignored` tag is necessary for Components-Generator.js
    */
-  public constructor(identifier?: MetadataIdentifier, overrides?: Record<string, MetadataOverrideValue>);
+  public constructor(identifier: MetadataIdentifier, overrides?: Record<string, MetadataOverrideValue>);
 
   /**
    * @param metadata - Starts as a copy of the input metadata.
    * @param overrides - Key/value map of extra values that need to be added to the metadata.
    *                    Will override values that were set by the input metadata.
    */
-  public constructor(metadata?: RepresentationMetadata, overrides?: Record<string, MetadataOverrideValue>);
+  public constructor(metadata: RepresentationMetadata, overrides?: Record<string, MetadataOverrideValue>);
```

Comment:
I remove these `?` such that the constructor with optional overrides as only argument would always unambiguously be https://github.com/solid/community-server/pull/491/files#diff-041823fa178d423e7bd04741d39a3817b6219f2b082109a2d723e5f6a917ec70R61